### PR TITLE
Internal: Fix the Add to iCall feature spellchecker quality control

### PR DESCRIPTION
**Pull request by robertragas**
## Problem
Currently the spellchecker is failing due to changes in the iCall feature.

## Solution
Make sure we add the words to the spellchecker definition as we cannot use any other wording for it, as they are ical attributes.

## How to test
- [ ] Wait for the tests to be green, mainly the spellcheck

### Before merge
- [ ] Code/peer review is completed
- [ ] All commit messages are [clear and clean](https://open-social.slite.com/app/docs/DnmermZDIx_0OQ). If applicable a rebase was performed
- [ ] All automated tests are green
- [ ] Functional/manual tests of the acceptance criteria are approved
- [ ] All acceptance criteria were met
- [ ] New features or changes to existing features are covered by tests, either unit (preferably) or behat
- [ ] Update path is tested. New hook_updates should respect update order, right naming convention and consider hook_post_update code
- [ ] Module can be safely uninstalled. Update/implement hook_uninstall and make sure that removed configuration or dependencies are removed/uninstalled
- [ ] This pull request has all required labels (team/type/priority)
- [ ] This pull request has a milestone
- [ ] This pull request has an assignee (if applicable)
- [ ] Any front end changes are tested on all major browsers
- [ ] New UI elements, or changes on UI elements are approved by the design team
- [ ] New features, or feature changes are approved by the product owner

### After merge
- [ ] Code is tested on all branches that it has been cherry-picked
- [ ] Update hook number might need adjustment, make sure they have the correct order
- [ ] The Drupal.org ticket(s) are updated according to this pull request status

## Release notes
Internal - quality check fix to get all the tests green

### CODE REVIEW

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

File: modules/social_features/social_event/modules/social_event_addtocal/src/Controller/AddToCalendarIcsController.php [hunk 1]
```diff
@@ -63,10 +63,10 @@ public function downloadIcs() {
     $dates = $this->request->get('dates');
     $timezone = $this->request->get('timezone');
 
-    // Generate DTSTAMP, which needs to be the time the ICS object created.
+    // Generate timestamp, which needs to be the time the ICS object created.
     // https://icalendar.org/iCalendar-RFC-5545/3-6-1-event-component.html
     $now = new \DateTime('now', new \DateTimezone('UTC'));
-    $dtstamp = $now->format('Ymd\THis\Z');
+    $timestamp = $now->format('Ymd\THis\Z');
```
Comment (Kingdutch) [inline modules/social_features/social_event/modules/social_event_addtocal/src/Controller/AddToCalendarIcsController.php:69]: We could also use `$dateTimeStamp` which CSpell should accept as well, but I think in this case timestamp is fine (even though we usually use that for UNIX timestamps and not YMDHis).

**Review by Kingdutch — APPROVED** [review]
Clear description of the additions to our dictionary <3

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3484-wgjbtf1cwthn1890swdzhoxqeiwo09ht.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64e3492fb5ff6a5a996119d2

**Review by bramtenhove — APPROVED** [review]

**Comment by robertragas** [comment]
Merged and cherry-picked to 11.10.3 and higher

**Pull request merged**